mypyc/codegen/emitfunc.py

```diff
@@ -88,8 +88,22 @@ def generate_native_function(fn: FuncIR,
             next_block = blocks[i + 1]
         body.emit_label(block)
         visitor.next_block = next_block
-        for op in block.ops:
+        ops = block.ops
```

Comment:
This could probably use a comment of what we are keeping track of here now that the loop is a lot more complex